pysoa/server/internal/event_loop.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+import threading
+
+
+def _loop_stop_callback(loop):
```

Comment:
Can you make this a `@staticmethod` on the `AsyncEventLoopThread` class instead of making it a function in the module? Would you even need the argument then, or could you just use `self.loop.stop()`?